### ISSUE TITLE
Future Patches

### ISSUE THREAD

**Issue by ccasalme** [ccasalme/Movie-MatchMaker#52]
# Checkbox: Seen it

- Quick fix and debug on the checkbox of "seen it" automatically checking again when we refresh

# Rating:

- Don't make it a requirement to rate a movie since if they have not watched it, they may not have an opinion of it yet (possibly an "if loop" situation)

![Screenshot 2024-11-19 at 5 02 57 PM](https://github.com/user-attachments/assets/4924317d-3c3d-4a9c-88bf-1f1de785205b)

![Screenshot 2024-11-19 at 5 04 04 PM](https://github.com/user-attachments/assets/1801a76d-18fd-449e-8ab5-bdc008d71c70)

**Comment by cpars** [comment]
Good idea for the "Rating" field. This will just require an edit to the index.html page. 

Note: remove _required_ from end of rating input tag
